tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
I guess that was my question, what would you transform them to anyways. If it's just meant as an otherwise useless injection point, then that makes sense.